Rakefile

```diff
@@ -31,17 +31,6 @@ task test: [
   :release_checks,
 ]
 
-desc "Run main 'test' task and report merged results to coveralls"
```

Comment:
This looks like you're undoing the latest modulesync changes I made in https://github.com/voxpupuli/modulesync_config/commit/8393b3267da937c83814ab24883ceba2fae5ce0e#diff-34d8b3ef11f42c86a299a165682095bd